Doc/library/stdtypes.rst

```diff
@@ -4114,6 +4114,12 @@ The constructors for both classes work the same:
 .. class:: set([iterable])
            frozenset([iterable])
 
+   Sets can be created by several means:
+
+   * Use a comma-separated list of elements within braces: ``{'jack', 'sjoerd'}``
```

Comment:
```suggestion
   * Using a comma-separated list of elements within braces: ``{'jack', 'sjoerd'}``
```

Reply:
Maybe a reminder that `{}` is the empty dictionary could be helpful here for novice users.